### PR TITLE
Hide only email address on contact page

### PR DESCRIPTION
**Pull request by gentlecat**
*Related forum topic: http://forums.musicbrainz.org/viewtopic.php?pid=32153.*

cc: @reosarevok, @mayhem

### CODE REVIEW

File: metabrainz/templates/index/contact.html [hunk 1]
```diff
@@ -52,36 +52,33 @@ <h1 class="page-title">Contact</h1>
       disallowed topics, we reserve the right to make fun of you.
     </em>
   </p>
+
+  <h3>E-mail us</h3>
   <button id="agree-btn" class="btn btn-default btn-success">
-    I agree that I will not contact you regarding the topics above
+    I agree with rules above, show me email address
```
Comment (reosarevok) [inline metabrainz/templates/index/contact.html]: Is "I won't contact you about these issues, show me the email address" too long? At least please add the articles ("the rules" / "the email") :)

File: metabrainz/templates/index/contact.html [hunk 2]
```diff
@@ -52,36 +52,33 @@ <h1 class="page-title">Contact</h1>
...
 
-    <h3>Mailing address</h3>
-    <p>
-      MetaBrainz Foundation<br />
-      3565 South Higuera St, Suite B<br />
-      San Luis Obispo, CA 93401<br />
-      United States
-    </p>
-  </div>
+  <h3>Mailing address</h3>
```
Comment (reosarevok) [inline metabrainz/templates/index/contact.html]: Unsure whether this bit should be hidden or shown. @mayhem? I guess we don't really read it anyway...

Comment (mayhem) [inline metabrainz/templates/index/contact.html]: Rrally, we only need to hide emails. Nothing else.

Comment (gentlecat) [inline metabrainz/templates/index/contact.html]: :car::dash:

File: metabrainz/templates/index/contact.html (hunk 1 above)
Comment (gentlecat) [inline metabrainz/templates/index/contact.html]: Fixed. Thanks.

**Pull request merged**